### ISSUE TITLE
snooze time entity missing after update to 2023.11b0

### ISSUE THREAD

**Issue by eikowagenknecht** [fuatakgun/eufy_security#963]
**Describe the bug**

After the update to HA 2023.11b0 I get the following 2 errors and the *_snooze_time entities are not available any more.

```
Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:507
Integration: Sensor (documentation, issues)
First occurred: 08:21:25 (1 occurrences)
Last logged: 08:21:25

Error adding entities for domain sensor with platform eufy_security
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 507, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 752, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1021, in add_to_platform_finish
    await self.async_internal_added_to_hass()
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 279, in async_internal_added_to_hass
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity sensor.au_carport_kamera_snooze_time cannot be added as the entity category is set to config

Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:367
Integration: Sensor (documentation, issues)
First occurred: 08:21:25 (1 occurrences)
Last logged: 08:21:25

Error while setting up eufy_security platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 367, in _async_setup_platform
    await asyncio.gather(*pending)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 507, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 752, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1021, in add_to_platform_finish
    await self.async_internal_added_to_hass()
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 279, in async_internal_added_to_hass
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: Entity sensor.au_carport_kamera_snooze_time cannot be added as the entity category is set to config
```

**Additional information**

Go to Settings -> System -> Repairs -> Click on 3 dots (...) -> System Information and get Version and Installation Type to below fields;
- Home Assistant Installation Type (OS - Supvervised - Core): Home Assistant OS 11.0
- Home Assistant Core Version: core-2023.11.0b0
- Eufy Security Add-on Version: eufy-security-ws (1.6.4)
- Eufy Security Integration Version: v7.7.1

**Comment by fuatakgun** [comment]
thanks, very useful, will fix this today

**Comment by fuatakgun** [comment]
Apparently sensor entities are not supposed to be marked as config, i still fix this

**Comment by fuatakgun** [comment]
fixed on version 7.7.2
